types/hapi/index.d.ts

```diff
@@ -1938,7 +1938,7 @@ export class Request extends Podium {
      * an object containing the query parameters.
      * TODO update with outcome of: https://github.com/hapijs/hapi/pull/3479
```

Comment:
Would be great to update this to the [latest docs](https://hapijs.com/api#request-properties) too:

> by default the object outputted from [node's URL parse()](https://nodejs.org/docs/latest/api/url.html#url_urlobject_query) method.  Might also be set indirectly via [request.setUrl](#requestseturlurl-striptrailingslash) in which case it may be a `string` (if `url` is set to an object with the `query` attribute as an unparsed string).